modules/rank.js

```diff
@@ -69,34 +89,35 @@ async function slashRankView(interaction) {
         "ain't interested in no XP gettin'.",
         "don't talk to me no more, so I ignore 'em."
       ];
-      await interaction.editReply(`**${member}** ${u.rand(snark)}\n(Try \`/rank track\` if you want to participate in chat ranks!)`);
+      await interaction.editReply(`**${member}** ${u.rand(snark)}\n(Try </rank track:${u.sf.commands.slashRank}> if you want to participate in chat ranks!)`).then(u.clean);
     }
   } catch (error) { u.errorHandler(error, interaction); }
 }
 
+/** @param {Augur.AugurClient} client */
 async function rankClockwork(client) {
   try {
-    const response = await u.db.user.addXp(active);
+    const response = await u.db.user.addXp([...active]);
     if (response.users.length > 0) {
       const ldsg = client.guilds.cache.get(u.sf.ldsg);
       for (const user of response.users) {
-        const member = ldsg.members.cache.get(user.discordId) ?? await ldsg.members.fetch(user.discordId).catch(u.noop);
+        const member = ldsg?.members.cache.get(user.discordId) ?? await ldsg?.members.fetch(user.discordId).catch(u.noop);
         if (!member) continue;
-
         try {
           // Remind mods to trust people!
-          if ((user.posts % 25 == 0) && !member.roles.cache.has(u.sf.roles.trusted) && !member.roles.cache.has(u.sf.roles.untrusted)) {
-            const modLogs = ldsg.channels.cache.get(u.sf.channels.modlogs);
-            modLogs.send({
+          const trustStatus = await u.db.user.fetchUser(member.id);
+          if ((user.posts % 25 == 0) && !member.roles.cache.has(u.sf.roles.trusted) && !trustStatus?.watching) {
```

Comment:
Property 'watching' does not exist on type 'UserRecord'.

Reply:
It should now